ENIGMAsystem/SHELL/Universal_System/instance_system_base.h

```diff
@@ -38,9 +38,9 @@ namespace enigma
 
   class temp_event_scope
   {
-    object_basic *oinst;
     inst_iter *oiter;
     object_basic* prev_other; //Should always be NULL, but just in case...
```

Comment:
Two more notes (no action required):
1. This comment of mine is wrong. This line of reasoning is the reason this bug exists. I think this mechanism was originally intended for instance creation scripts, and only later came to be used in regular creation events. All that needs to happen for this to not be `NULL` is for `instance_create` to be called in a collision event.
2. I believe we call the variable below `niter`, elsewhere (similar to `ninst`). This naming convention is largely unhelpful, though. Even just saying "new" instead would have been better....